package.json

```diff
@@ -24,9 +24,9 @@
     "url": "https://github.com/slack-samples/bolt-js-assistant-template/issues"
   },
   "dependencies": {
-    "@huggingface/inference": "^4.7.1",
```

Comment:
👁️‍🗨️ question: Are we wanting to keep Hugging Face as an optional provider kept in dependencies? IMHO asking folks to install this is not a great setup step and we should maintain this logic or remove it altogether in from samples-

Reply:
📚 ramble: We might also want to consider existing tutorials that hint at these implementations:

🔗 https://docs.slack.dev/tools/bolt-js/tutorials/code-assistant

🧠 question: Perhaps @lukegalbraithrussell has thoughts on these writings also?